language_detector/main.py

```diff
@@ -1,9 +1,21 @@
-"""This is the entry point of the program."""
 
-from .languages import LANGUAGES
 
+from .languages import LANGUAGES
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    lan_count = {l['name']:0 for l in languages}  #create dictionary to count words with
```

Comment:
Even though we love well documented code, being too verbose while documenting could be controversial. For example, in this line it's quite obvious that we are creating a dictionary. It's probably not necessary to comment that.